react/components/Helpers.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+export const Spacer = ({vertical=0, horizontal=0, index=1}) => (
```

Comment:
Why `vertical` and `horizontal` naming? I think `height` and `width` could be a better fit for understanding, since these are the css properties this props will be bound to.